zerver/webhooks/github/tests.py

```diff
@@ -302,6 +302,21 @@ def test_check_run(self) -> None:
 """.strip()
         self.send_and_test_stream_message('check_run__completed', expected_topic, expected_message)
 
+    def test_team_edited_description(self) -> None:
+        expected_topic = "Team Testing"
+        expected_message = """The team description has been changed to: ```A temporary team so that I can get some webhook fixtures!```."""
+        self.send_and_test_stream_message('team__edited_description', expected_topic, expected_message)
+
+    def test_team_edited_name(self) -> None:
+        expected_topic = "Team Testing Team"
+        expected_message = """Formerly named `Testing`."""
```

Comment:
Let's just say the whole thing; "Team X was renamed to Y."  It's OK to repeat the name from the topic for clarity here.